### ISSUE TITLE
`unused-void-attribute` does not understand formation chaining

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#573]
Consider this EO snippet:

```eo
# This unit test is supposed to check the functionality of the corresponding object.
[] > foo
  boom > @
    filteredi.
      list
        * true false true
      v.not > [v i] # <-- here
    * false
```

Here, `v` and `i` are chained into another formation, therefore they are _used_, however we are receiving this complaint:

```text
The void attribute "v" is unused anywhere in the anonymous object
The void attribute "i" is unused anywhere in the anonymous object
```

**Comment by h1alexbel** [comment]
@yegor256 @maxonfjvipon should we allow void attributes to be "chained" to other formations? WDYT?

**Comment by h1alexbel** [comment]
my mistake, this ticket is incorrect
